_maps/shuttles/shiptest/cargotide.dmm

```diff
@@ -1841,7 +1852,7 @@ eV
 vi
 rV
 RR
-tZ
```

Comment:
this looks like a mistake, did you accidentally remove something?

Reply:
meant to reply, thats just SDMM making that tile into one with a window

Reply:
ok sounds good